server.go

```diff
@@ -2543,6 +2577,42 @@ func (s *Server) processor(client MessagingClient, done chan struct{}) {
 	}
 }
 
+// flusher sends raw point data in batches to the broker by topic.
+func (s *Server) flusher(client MessagingClient, done chan struct{}) {
+	ticker := time.NewTicker(s.FlushInterval)
```

Comment:
Ah yes -- a subtle, but important point. I used `After` in retention policy enforcement, but the RP check-intervals are very large there relative to milliseconds, so I think the RP code is fine.